UPGRADING.rst

```diff
@@ -10,6 +10,18 @@ branch that does not have the listed changes, the steps would need to be
 reverted. This is all fairly informal and loosely defined. Hopefully we won't
 have too many entries in this file.
 
+#1673 Ensure Lambda package hash is deterministic
+=================================================
+
+#. If you haven't yet, install Python3.8.2.
+
+#. Recreate your virtual environment with Python3.8.2::
+
+    make virtualenv
+    make requirements
+
+Note: You may need to update your interpreter settings if you use PyCharm.
```

Comment:
I'm surprised by this since the venv path stays the same. Be more specific: what do I need to do?

Reply:
Well for me, PyCharm still listed the interpreter as 3.6 even though it was using 3.8. I Don't think there was actually any problem, but the inaccurate number bothered me, so I deleted and removed the interpreter from the list of options.

Since it's not strictly necessary, I'll just remove this note.

Reply:
> Since it's not strictly necessary, I'll just remove this note.

Please do so.